ocean_lib/models/datatoken_base.py

```diff
@@ -175,7 +176,12 @@ def get_typed(config, address):
         except Exception:
             template_id = 1
 
-        return datatoken if template_id == 1 else Datatoken2(config, address)
+        if template_id == 1:
+            return Datatoken1(config, address)
+        if template_id == 2:
+            return Datatoken2(config, address)
+        if template_id == 3:
+            return Datatoken3(config, address)
```

Comment:
Change to:
```python
        if template_id == 1:
            return Datatoken1(config, address)
        elif template_id == 2:
            return Datatoken2(config, address)
        elif template_id == 3:
            return Datatoken3(config, address)
        else:
            return ValueError(template_id)
```

Why: more pythonic if/else. And, catch error when unknown input arg.